css/style.css

```diff
@@ -1,11 +1,163 @@
 /* Simple CSS Reset */
 * {
-  margin: 0;
-  padding: 0;
+  	margin: 0;
+  	padding: 0;
+}
+body {
+	color: #f0efef;
+  font-size: 26px;
+  line-height: 1.4;
+  color: #606161;
+  font-family: "Open Sans", Helvetica, Arial, sans-serif;
+  font-weight: 300;
+}
+a {
+	text-decoration: none;
 }
-
 header {
   height: 700px;
   background: url("../images/header_bg.jpg") no-repeat center top;
   background-size: cover;
+  }
+h1{
+	color: white;
+	padding-left: 0px;
+	padding-top: 20px;
+	font-size: 24px;
+	font-weight: 800; 
+	margin-right: 600px;
+}
+h2{
+	margin-top: 100px;
+	text-align: center;
+	color: white;
+	font-weight: 600;
+}
+h3{
+	font-weight: 600;
+	font-size: 20px;
+	color: #033048;
+	padding-bottom: 17px;
+}
+h4{
+	font-size: 18px;
+	color: white;
+	font-weight: 600;
+}
+.button {
+	margin-top: 39px;
```

Comment:
This margin top is not working because `<a>` tags are inline elements. Apply display block to make it a block element. Then give the element a width to control how wide it is.